src/GovUk.Education.ExploreEducationStatistics.Data.Services/TableBuilderService.cs

```diff
@@ -111,4 +154,11 @@ await _userService.MatchesPolicy(subject, CanViewSubjectData))
             return new ForbidResult();
         }
     }
+
+    public class TableBuilderOptions
+    {
+        public const string TableBuilder = "TableBuilder";
```

Comment:
What's the convention here?  Why do we need this?

Reply:
It's just the section name expected in the configuration JOSN data.

I suppose it's  not really essential.

Instead of 

```
services.Configure<TableBuilderOptions>(Configuration.GetSection("TableBuilder"));
```

we do

```
services.Configure<TableBuilderOptions>(Configuration.GetSection(TableBuilderOptions.TableBuilder));
```

Since we do this in two places (Data.API and Admin) the constant kind of makes sense to tie them together.

See the following section of the documentation where I got this convention from:

https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-5.0#bind-hierarchical-configuration

where it says

> An alternative approach when using the options pattern is to bind the Position section and add it to the dependency injection service container. In the following code, PositionOptions is added to the service container with Configure and bound to configuration

Their example is

```
services.Configure<PositionOptions>(Configuration.GetSection(PositionOptions.Position));
```